mod_reforged/hooks/items/armor/armor.nut

```diff
@@ -0,0 +1,10 @@
+::Reforged.HooksMod.hook("scripts/items/armor/armor", function(q) {
```

Comment:
The addition of this vanilla fix should be a separate commit so that it can be reverted separately later if MSU adds this fix.

Reply:
It was already a separate commit, unless I confuse something